src/Assert/Assert.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Assert;
+
+use Wikimedia\Assert\Assert as WikimediaAssert;
+use Wikimedia\Assert\ParameterAssertionException;
+
+/**
+ * Provides functions to assure values meet certain preconditions relevant
+ * in Wikibase.
+ *
+ * @see Wikimedia\Assert\Assert
+ *
+ * @since 3.7
+ *
+ * @license GPL-2.0+
+ */
+class Assert {
+
+	/**
+	 * @param string $value
+	 * @param string $name
+	 *
+	 * @throws ParameterAssertionException if $value is not an allowable repository name.
+	 */
+	public static function parameterIsAllowableRepositoryName( $value, $name ) {
+		if ( !self::isAllowableRepositoryName( $value ) ) {
+			throw new ParameterAssertionException( $name, 'must be a string not including colons' );
+		}
+	}
+
+	/**
+	 * @param array $value
+	 * @param string $name
```

Comment:
Well, I kinda re-used the interface of wikimedia/Assert: https://github.com/wikimedia/Assert/blob/master/src/Assert.php
$name is a name of the parameter. It is passed over to the exceptions thrown.